### PR TITLE
Changed: Suppress result point output for big groups (lines)

### PR DESCRIPTION
**Pull request by kmokstad**
Triviality. Just tired of the long output of the result lines after each mesh refinement..

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this with downstreams ifem-openfrac=19 please

File: src/SIM/SIMoutput.C [hunk 1]
```diff
@@ -216,8 +216,12 @@ void SIMoutput::preprocessResPtGroup (std::string& ptFile, ResPointVec& points)
     else if ((p->inod = pch->evalPoint(p->u,p->u,p->X)) < 0)
       p = points.erase(p);
     else
+      (p++)->npar = pch->getNoParamDim();
+  }
+
+  if (points.size() < 5 || msgLevel > 1)
+    for (ResPointVec::const_iterator p = points.begin(); p != points.end(); ++p)
```
Comment (akva2) [inline src/SIM/SIMoutput.C]: range based for while at it ? unless there is some iterator usage hiding in lines not shown..

Comment (kmokstad) [reply src/SIM/SIMoutput.C]: Not really, except for then have to change all `->` into `.` Can do that of course

**Comment by kmokstad** [comment]
jenkins build this with downstreams ifem-openfrac=19 please

**Pull request merged**